test/Unit/PowerShell/PowerShellManagerTests.cs

```diff
@@ -159,5 +159,79 @@ public void RegisterAndUnregisterFunctionMetadataShouldWork()
             manager.UnregisterFunctionMetadata();
             Assert.Empty(FunctionMetadata.OutputBindingCache);
         }
+
+        [Fact]
+        public void ProfileShouldWork()
+        {
+            var logger = new ConsoleLogger();
+            var manager = new PowerShellManager(logger);
```

Comment:
Just curious, is it possible to reuse a `PowerShellManager` in those tests?

Reply:
reused using a static manager and logger